webpack.config.minified.js

```diff
@@ -0,0 +1,15 @@
+var webpack = require('webpack');
+var _ = require('lodash');
+
+module.exports = _.merge(require('./webpack.config.js'), {
```

Comment:
Switch to

```js
module.exports = _.merge({}, require('./webpack.config.js'), {
```

If you end up using 2+ configurations in the same process (aka, what often happens in gulp-land...)

@alexlande -- Want to ticket that out?